docs/faq.md

```diff
@@ -0,0 +1,310 @@
+
+
+*What does WIP mean?*
+
+WIP means that we don't try to be very backward-compatible at the moment. At some point we may change wallet storage format for example, and you would need to migrate using some script or create wallets from scratch. In this case, we would provide migration scripts.
+
+
+*What's the difference between specter-desktop and specter-DIY?*
+
+Specter-desktop is only a wrapper (watch-only) around Bitcoin Core wallet functionality, mostly, which uses your full node. Bitcoin Core tracks addresses, UTXO (unspent transaction outputs) and composes PSBT (partially-signed bitcoin transactions). But all wallets are watch-only.
+
+Whereas, Specter-DIY is a do-it-yourself hardware wallet from off the shelf components, that signs and broadcasts transactions using QR codes that forgets your private keys when powered off.
+
+
+*Do you need Specter to have a wallet on your node for it to even start? I was hoping to try this without using the core wallet at all.*
+
+You don't need private keys in Bitcoin Core, but you need wallets to be enabled `disablewallet=0` in your `bitcoin.conf` file.
+
+*I make unsigned transactions from my cold storage using a watching-only Electrum wallet. I use public servers instead of my own node because doing it "right" is too complicated for me. Specter may be an ideal alternative if it will connect to my **headless bitcoind node**. Will this be possible?*
+
+Yes, this is the plan - to use a HW wallet like Coldcard/Trezor with Specter DIY, with a user-friendly multisig Specter desktop app, which is connected to your own node for better privacy.
+
+
+*What is the practical difference of using PSBT (partially signed bitcoin transaction) with multisig vs. just signing the raw multisig transaction normally?*
+
+It works better with hardware wallets: https://github.com/bitcoin-core/HWI  
+
+
+### USAGE
+
+*How do I run the app?*
+
+You should be able to view it in a browser at: 127.0.0.1:25441/
+If not, see [Troubleshoot](https://github.com/cryptoadvance/specter-desktop/new/master/docs#troubleshoot)
+
+
+*What do I need to do in order to create a multisig wallet?*
+
+First you need to “add devices” that store keys for the wallet. Once you add at least two devices, then the multsig option will become available to you in the UI.
+
+*Can I use Bluewallet with Specter DIY?*
+
+Yes you can use BlueWallet in watch-only mode and sign with Specter DIY.
+
+*Which HWW's are supported?*
+
+Any HWW with HWI, including USB HWW's (Coldcard, Trezor, Ledger, KeepKey, BitBox, etc.)
+
+*Can this also work with external nodes like Casa, MyNode, and Raspilitz?* 
+
+Absolutely, as well as any other DIY bitcoin full, or pruned, node!
+
+
+*Can I use Tor?*
+
+Yes there is a way to access specter-desktop over Tor from outside, here is the [doc](https://github.com/cryptoadvance/specter-desktop/blob/master/docs/tor.md)
+
```

Comment:
Beware that it's not practical yet to sign transactions via Tor:
* specter-diy needs the camera which is not available in the tor-browser (yet)
* YOu could use HWI-wallets but you would need to plug the wallet into the machine where specter-desktop is running on. This is usually not the use-case you're looking for when using tor.